mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -114,8 +114,13 @@ fun ResponseBody.toBitmap(targetSize: Int, enforceSize: Boolean = false): Bitmap
         }
     }
 
+    return bitmapToSvg(string(), targetSize)
+}
+
+@Throws(IOException::class)
+fun bitmapToSvg(svgString: String, targetSize: Int): Bitmap {
```

Comment:
The naming is off. The method name implies a bitmap-to-SVG conversion, but it's the other way around.
Besides that, I'd suggest making this `fun InputStream.asSvgToBitmap(targetSize: Int): Bitmap` ... the current method isn't an extension function, which makes it a bit mis-placed in that file ;-) and looking at how the method is used, we have an `InputStream` readily available, so why do the extra string conversion?

Reply:
Done